cs-utilities/src/main/java/io/cloudslang/content/utilities/actions/FindTextInPdf.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (c) Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+package io.cloudslang.content.utilities.actions;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import io.cloudslang.content.constants.ReturnCodes;
+import io.cloudslang.content.utilities.services.PdfParseService;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType.COMPARE_EQUAL;
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType.ERROR;
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType.RESOLVED;
+import static io.cloudslang.content.constants.BooleanValues.FALSE;
+import static io.cloudslang.content.constants.OutputNames.*;
+import static io.cloudslang.content.constants.ResponseNames.FAILURE;
+import static io.cloudslang.content.constants.ResponseNames.SUCCESS;
+import static io.cloudslang.content.utilities.entities.constants.Descriptions.InputsDescription.*;
+import static io.cloudslang.content.utilities.entities.constants.Descriptions.OperationDescription.FIND_TEXT_IN_PDF_OPERATION_DESC;
+import static io.cloudslang.content.utilities.entities.constants.Descriptions.OutputsDescription.*;
+import static io.cloudslang.content.utilities.entities.constants.Descriptions.ResultsDescription.FAILURE_DESC;
+import static io.cloudslang.content.utilities.entities.constants.Descriptions.ResultsDescription.SUCCESS_DESC;
+import static io.cloudslang.content.utilities.entities.constants.Inputs.*;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static io.cloudslang.content.utils.OutputUtilities.getSuccessResultsMap;
+import static java.lang.String.valueOf;
+import static org.apache.commons.lang3.BooleanUtils.toBoolean;
+import static org.apache.commons.lang3.StringUtils.countMatches;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+/**
+ * Created by marisca on 7/11/2017.
+ */
+public class FindTextInPdf {
+
+    /**
+     * This operation checks if a text input is found in a PDF file.
+     *
+     * @param text       The text to be searched for in the PDF file.
+     * @param ignoreCase Whether to ignore if characters of the text are lowercase or uppercase.
+     *                   Valid values: "true", "false". For any other value the ignoreCase will be set to "false".
```

Comment:
usually only "true" or "false" are allowed, other values are considered and error ( invalid user input, think about  "si" or "\t true" will both be considered as "false" an the user will not know why the behavior is not the expected one)